app/routes/transfer.py

```
@@ -19,143 +16,91 @@ class SEPARequest(BaseModel):
     db.add(tx)
     db.commit()
-    if reason:
-        create_aml_alert(db, current_user.id, reason, risk_level, client_ip, tx.id)
-    return {"message": "ACH transfer sent", "reference": ref, "status": tx.status}
+    return {"message": "ACH transfer sent", "reference": ref}
 
 @router.post("/internal")
 def internal_transfer(
     req: InternalTransferRequest,
-    request: Request,
     current_user: User = Depends(get_current_user),
```

Ruff (0.15.6)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)